config/example/node.yaml

```diff
@@ -1,159 +1,159 @@
+    address: "NcpJzXcSDrh5CCizf4K9Ro6w4t59J5LKzz"  # address of a NEO account in the wallet; ignored if key is presented
+    password: "password"  # password for a NEO account in the wallet; ignored if key is presented
+  addresses:  # addresses for bootstrapping in the Network map
     - s01.neofs.devenv:8080
     - /dns4/s02.neofs.devenv/tcp/8081
     - grpc://127.0.0.1:8082
     - grpcs://localhost:8083
   attribute_0: "Price:11"
   attribute_1: UN-LOCODE:RU MSK
-  relay: true
-  persistent_state:
+  relay: true  # (not)register Node in the Network map on startup
+  persistent_state:  # path to store Node state persistently
     path: /state
 
 grpc:
   num: 2
   0:
-    endpoint: s01.neofs.devenv:8080
+    endpoint: s01.neofs.devenv:8080  # endpoint for gRPC server
     tls:
-      enabled: true
-      certificate: /path/to/cert
-      key: /path/to/key
+      enabled: true  # use TLS for a gRPC connection
+      certificate: /path/to/cert  # path to TLS certificate
+      key: /path/to/key  # path to TLS key
 
   1:
-    endpoint: s02.neofs.devenv:8080
+    endpoint: s02.neofs.devenv:8080  # endpoint for gRPC server
     tls:
-      enabled: false
+      enabled: false  # use TLS for a gRPC connection
 
 control:
-  authorized_keys:
+  authorized_keys:  # list of raw hex public keys that have rights to use the Control Service
     - 035839e45d472a3b7769a2a1bd7d54c4ccd4943c3b40f547870e83a8fcbfb3ce11
     - 028f42cfcb74499d7b15b35d9bff260a1c8d27de4f446a627406a382d8961486d6
-  grpc:
+  grpc:  # endpoint that are listened by the Control Service
     endpoint: 127.0.0.1:8090
 
-contracts:
+contracts:  # side chain NEOFS contract script hashes
   balance: 5263abba1abedbf79bb57f3e40b50b4425d2d6cd
   container: 5d084790d7aa36cea7b53fe897380dab11d2cd3c
   netmap: 0cce9e948dca43a6b592efe59ddb4ecb89bdd9ca
   reputation: 441995f631c1da2b133462b71859494a5cd45e90
   proxy: ad7c6b55b737b696e5c82c85445040964a03e97f
 
 morph:
-  dial_timeout: 30s
-  disable_cache: true
-  rpc_endpoint:
+  dial_timeout: 30s  # timeout for side chain NEO RPC client connection
+  disable_cache: true  # do not use TTL cache for side chain GET operations
+  rpc_endpoint:  # side chain NEO RPC endpoints; are shuffled and used one by one until the first success
     - https://rpc1.morph.fs.neo.org:40341
     - https://rpc2.morph.fs.neo.org:40341
-  notification_endpoint:
+  notification_endpoint:  # side chain NEO RPC notification endpoints
     - wss://rpc1.morph.fs.neo.org:40341/ws
     - wss://rpc2.morph.fs.neo.org:40341/ws
   notary:
-    deposit_amount: 100000001
-    deposit_duration: 1001
+    deposit_amount: 100000001  # notary deposit GAS amount
+    deposit_duration: 1001  # notary deposit expiration in blocks
 
 mainchain:
-  dial_timeout: 30s
-  rpc_endpoint:
+  dial_timeout: 30s  # timeout for main chain NEO RPC client connection
+  rpc_endpoint:  # main chain NEO RPC endpoints; are shuffled and used one by one until the first success
```

Comment:
same here

Reply:
hmmm

in fact, node does not use a connection to the main chain at all after the last(`v0.25.0`) release, so marked that section as deprecated